Disable ALSA and FluidSynth (MIDI) support : use SDL for PCM/CD/Music support.
See jfaudiolib/src/drivers.c for the exact role of each driver.

--- jfaudiolib/Makefile.shared.orig	2013-06-01 00:16:56 UTC
+++ jfaudiolib/Makefile.shared
@@ -18,12 +18,12 @@ else
   JFAUDIOLIB_HAVE_SDL=1
   JFAUDIOLIB_LDFLAGS+= $(shell pkg-config --libs sdl)
  endif
- ifeq (yes,$(shell pkg-config --exists alsa && echo yes))
-  JFAUDIOLIB_HAVE_ALSA=1
-  JFAUDIOLIB_LDFLAGS+= $(shell pkg-config --libs alsa)
- endif
- ifeq (yes,$(shell pkg-config --exists fluidsynth && echo yes))
-  JFAUDIOLIB_HAVE_FLUIDSYNTH=1
-  JFAUDIOLIB_LDFLAGS+= $(shell pkg-config --libs fluidsynth)
- endif
+# ifeq (yes,$(shell pkg-config --exists alsa && echo yes))
+#  JFAUDIOLIB_HAVE_ALSA=1
+#  JFAUDIOLIB_LDFLAGS+= $(shell pkg-config --libs alsa)
+# endif
+# ifeq (yes,$(shell pkg-config --exists fluidsynth && echo yes))
+#  JFAUDIOLIB_HAVE_FLUIDSYNTH=1
+#  JFAUDIOLIB_LDFLAGS+= $(shell pkg-config --libs fluidsynth)
+# endif
 endif
